tests/query_interface_test.js

```diff
@@ -0,0 +1,587 @@
+require('./helper');
+
+var expect = require('chai').expect;
+var Sequelize = require('..');
+
+const DataTypes = Sequelize.DataTypes;
+const _ = require('lodash');
+
+describe('QueryInterface', () => {
+  beforeEach(function() {
+    this.sequelize.options.quoteIdenifiers = true;
+    this.queryInterface = this.sequelize.getQueryInterface();
+  });
+
+  afterEach(async function() {
+    await this.queryInterface.dropAllSchemas();
+  });
+
+  describe('dropAllSchema', () => {
+    it('should drop all schema', async function() {
+      await this.queryInterface.dropAllSchemas({
+        skip: [this.sequelize.config.database]
+      });
+      const schemaNames = await this.queryInterface.showAllSchemas();
+      await this.queryInterface.createSchema('newSchema');
+      const newSchemaNames = await this.queryInterface.showAllSchemas();
+      expect(newSchemaNames).to.have.length(schemaNames.length + 1);
+      await this.queryInterface.dropSchema('newSchema');
+    });
+  });
+
+  // Skip only local, on CI it works
+  describe.skip('showAllTables', () => {
+    it('should not contain views', async function() {
+      async function cleanup() {
+        // NOTE: The syntax "DROP VIEW [IF EXISTS]"" is not part of the standard
+        // and might not be available on all RDBMSs. Therefore "DROP VIEW" is
+        // the compatible option, which can throw an error in case the VIEW does
+        // not exist. In case of error, it is ignored.
+        try {
+          await this.sequelize.query('DROP VIEW V_Fail');
+        } catch (error) {
+          // Ignore error.
+        }
+      }
+      await this.queryInterface.createTable('my_test_table', { name: DataTypes.STRING });
+      await cleanup.call(this);
+      await this.sequelize.query('CREATE VIEW V_Fail AS SELECT 1 Id');
+      let tableNames = await this.queryInterface.showAllTables();
+      await cleanup.call(this);
+      if (tableNames[0] && tableNames[0].tableName) {
+        tableNames = tableNames.map(v => v.tableName);
+      }
+      expect(tableNames).to.deep.equal(['my_test_table']);
+    });
+  });
+
+  // solved in another PR
+  describe.skip('renameTable', () => {
+    it('should rename table', async function() {
+      await this.queryInterface.createTable('my_test_table', {
+        name: DataTypes.STRING
+      });
+      await this.queryInterface.renameTable('my_test_table', 'my_test_table_new');
+      let tableNames = await this.queryInterface.showAllTables();
+      expect(tableNames).to.contain('my_test_table_new');
+      expect(tableNames).to.not.contain('my_test_table');
+    });
+  });
+
+  // already working
```

Comment:
Is this comment necessary?

Reply:
no, it was a self note to keep track of the tests